packages/celery-library/tests/conftest.py

```diff
@@ -33,11 +39,40 @@
 ]
 
 
+_logger = logging.getLogger(__name__)
+
+
 class FakeAppServer(BaseAppServer):
-    async def lifespan(self, startup_completed_event: threading.Event) -> None:
+    def __init__(self, app: Celery, settings: CelerySettings):
```

Comment:
I am wondering a bit why in this celery package we are not testing with the real celery? we have all the infrastructure to put up rabbit/redis or whatever you want. this package being small and only running against celery would be perfect.

Reply:
Celery is real, this just create a fake wrapper for any AppServer.